jinja2/base.html

```diff
@@ -42,7 +42,9 @@
   {% stylesheet 'locale-%s' % LANG %}
 
   {% block site_analytics %}
-    {% javascript 'dnt-helper' %}
+    <script>
+      {% include "js/libs/mozilla.dnthelper.min.js" %}
```

Comment:
We could move this inside ``includes/google_analytics.html``, and then there would be one inline script block in the ``<head>``.

Reply:
Ah yes, that is a good idea. 👍